view/frontend/web/js/components/BaseButton.vue

```diff
@@ -1,22 +1,53 @@
 <template>
-  <button :type="buttonType" :title="buttonTitle">
-    {{ text }}
+  <button
+    class="button"
+    :type="buttonType"
+    :title="buttonTitle"
+    :disabled="isBlocked"
```

Comment:
maybe we can set disabled button if prop withLoader is true?

Reply:
prop `withLoader` is not a loader state. It's like a variant for the button. You can set button without or with loader. So I can't disable the button when withLoader is true b/c it will be disabled it on start